java/vector/src/main/java/org/apache/arrow/vector/util/DecimalUtility.java

```diff
@@ -95,6 +95,24 @@ public static boolean checkPrecisionAndScale(BigDecimal value, int vectorPrecisi
     return true;
   }
 
+  /**
+   * Check that the decimal scale equals the vectorScale and that the decimal precision is
+   * less than or equal to the vectorPrecision. If not, then an UnsupportedOperationException is
+   * thrown, otherwise returns true.
+   */
+  public static boolean checkPrecisionAndScale(int decimalPrecision, int decimalScale, int vectorPrecision,
```

Comment:
can be void..nothing to do with return status..

Reply:
tried to keep it consistent with the method above it.